pycqed/instrument_drivers/physical_instruments/QuTech_ControlBox_v3.py

```diff
@@ -38,6 +38,15 @@ def add_params(self):
                            get_cmd=self._do_get_trigger_source,
                            vals=vals.Enum('internal', 'external',
                                           'mixed'))
+        self._pulse_queue_state = defHeaders.pulse_queue_states[0]
+        self.add_parameter('pulse_queue_state',
```

Comment:
This parameter may benefit from a docstring, explaining what this paramter means. I have some sort of idea but it is rather vague. I think that the name (depending on what it does) is still a good name. 

A docstring can be added quite simply by using "docstring = ". I've added a little snippet from another instrument with an example of how to do that. 

```python 
        self.add_parameter('kernel_list',
                           initial_value=[],
                           vals=vals.Anything(),
                           parameter_class=ConfigParameter,
                           docstring='List of external kernels to be loaded')
```